### ISSUE TITLE
How to balance the category in mmdetection

### ISSUE THREAD

**Issue by gfjiangly** [open-mmlab/mmdetection#996]
For example, some categories have only a dozen objects, while others have tens of thousands of objects. How to balance the category in mmdetection? Is there a configuration parameter for adding balance class weights for cross entropy loss in mmdetection?

**Comment by hellock** [comment]
It is currently not available by just modifying the config file. You may need to modify the arguments of the [loss function](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/losses/cross_entropy_loss.py#L11) manually.
